ETC.tsv

```diff
@@ -97,109 +97,109 @@ ETC_20150317_000096	$Although their bright color makes them pretty visible, it's
 ETC_20150317_000097	$Yellow Griba
 ETC_20150317_000098	$Large Red Griba
 ETC_20150317_000099	$Big Griba
-ETC_20150317_000100	Someone once described this monster as hard to block and hard to eat.
+ETC_20150317_000100	$Someone once described this monster as hard to block and hard to eat.
 ETC_20150317_000101	$Big Blue Griba
-ETC_20150317_000102	A large loathsome creature that is intentionally hunted.
+ETC_20150317_000102	$Large, strong, and tough monster which is a reluctant prey of the hunters
 ETC_20150317_000103	$Green Big Griba
 ETC_20150317_000104	$Red Big Griba
 ETC_20150317_000105	$Old Kepa
-ETC_20150317_000106	Kepas have aged the Pokusaku flower through constantly leaching it. Caution is advised as it emits a strong spicy scent.
-ETC_20150317_000107	Gelsvas Old Kepa
+ETC_20150317_000106	$While expending its strength to flower, the Kepa ages quickly. Be careful of its pungent scent, which has gotten stronger with age.
+ETC_20150317_000107	$Gelsvas Old Kepa
 ETC_20150317_000108	$Black Old Kepa
 ETC_20150317_000109	$Red Old Kepa
-ETC_20150317_000110	Its dilapidated color from aging through life only substantiates its danger.
+ETC_20150317_000110	$Its dilapidated color from aging through life only substantiates its danger.
 ETC_20150317_000111	$Large Old Kepa
 ETC_20150317_000112	$Rafflesia
 ETC_20150317_000113	$Green Rafflesia
 ETC_20150317_000114	$Black Rafflesia
-ETC_20150317_000115	The leaves of this plant are stronger than leather.
+ETC_20150317_000115	$The leaves of this plant are stronger than leather.
```

Comment:
"This plant's leaves are stronger than leather."

Black Rafflesia made the subject of the sentence.